src/webportal/server/config/logger.js

```diff
@@ -15,53 +15,50 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-
 // module dependencies
 const util = require('util');
 const winston = require('winston');
 const config = require('./index');
 
-
 const logTransports = {
   console: new winston.transports.Console({
     json: false,
     colorize: true,
     timestamp: () => new Date().toISOString(),
-    formatter: (options) => {
+    formatter: options => {
```

Comment:
do you disable [arrow-parens](https://eslint.org/docs/rules/arrow-parens) in eslint?

Reply:
This is the [default value](https://prettier.io/docs/en/options.html#arrow-function-parentheses) of prettier. Last discussion meeting we went through this rule and agree with the default. If you think it is mandatory, I can add them back.

Reply:
ok